Tests/Clappr_Tests/Classes/Base/Layers/MediaControlLayerTests.swift

```diff
@@ -0,0 +1,39 @@
+import Quick
+import Nimble
+@testable import Clappr
+
+class MediaControlLayerTests: QuickSpec {
+    override func spec() {
+        describe(".MediaControlLayer") {
+            context("When a MediaControl is attached") {
+                it("resizes to match the layer bounds") {
+                    let mediaControlView = UIView(frame: .zero)
+                    let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+                    let layer = MediaControlLayer(frame: frame)
+
+                    layer.attachMediaControl(mediaControlView)
+                    layer.layoutIfNeeded()
+
+                    expect(mediaControlView.center).to(equal(layer.center))
+                    expect(mediaControlView.bounds.size).to(equal(layer.bounds.size))
+
+                }
+                context("and the view size changes") {
+                    it("resizes to match the view bounds") {
+                        let smallFrame = CGRect(x: 0, y: 0, width: 50, height: 50)
+                        let bigFrame = CGRect(x: 0, y: 0, width: 100, height: 100)
+                        let mediaControlView = UIView(frame: .zero)
+                        let mediaControlLayer = MediaControlLayer(frame: smallFrame)
+
+                        mediaControlLayer.attachMediaControl(mediaControlView)
+                        mediaControlLayer.layoutIfNeeded()
```

Comment:
Same as above. Let me know what you think about the suggestion.

Reply:
This call is as redundant as the one on the above comment.

But just to offer some context, these calls to `layoutIfNeeded` are necessary to force the view size update. 

We're calling `translatesAutoresizingMaskIntoConstraints = false` in the implementation code to avoid conflicts in our constraints. But this actually invalidates direct frame modifications, so during testing, we need to call `layoutIfNeeded` to make AutoLayout do its magic.

Reply:
Same answer as above.